src/main/java/io/getunleash/util/UnleashURLs.java

```diff
@@ -10,12 +10,15 @@ public class UnleashURLs {
     private final URL clientMetricsURL;
     private final URL clientRegisterURL;
 
+    private final URL featuresURL;
+
     public UnleashURLs(URI unleashAPI) {
         try {
             String unleashAPIstr = unleashAPI.toString();
             fetchTogglesURL = URI.create(unleashAPIstr + "/client/features").normalize().toURL();
             clientMetricsURL = URI.create(unleashAPIstr + "/client/metrics").normalize().toURL();
             clientRegisterURL = URI.create(unleashAPIstr + "/client/register").normalize().toURL();
+            featuresURL = URI.create(unleashAPIstr + "/client/register").normalize().toURL();
```

Comment:
Do we need another URL here? Isn't the new data already coming from the fetchTogglesURL?

Small nit, but an URL named featuresURL that's pointing to a register endpoint seems like a smell.

Reply:
please remove this URL before merging @andreas-unleash

Reply:
my bad, it was removed 👍🏼